jameda_interface_config.js

```diff
@@ -364,8 +363,4 @@ if (isBrowserFirefoxOrSafari() === 'Safari') {
     ];
 }
 
-if (Boolean(window.jameda) && Boolean(window.jameda.inst) && Boolean(window.jameda.inst.brandLogoPath)) {
```

Comment:
I guess it's better to keep default_logo_url as fallback in case window.jameda.inst.brandLogoPath doesn't exist.

Reply:
Always providing a fallback would be a nice, future-proof solution in case another frontend app would use it in the future.